stellar-dotnet-sdk-xdr/generated/AccountEntry.cs

```diff
@@ -1,5 +1,6 @@
-﻿// Automatically generated by xdrgen
+// Automatically generated by xdrgen
```

Comment:
This line changing is removing the UTF-8 BOM characters from the file, essentially making the file us-ascii. xdrgen seems to generate us-ascii files, and from what I can see some other files in this repo are us-ascii too. At some point an IDE must have caused the XDR generated files to become UTF-8?